test/realm_test.dart

```diff
@@ -25,14 +25,14 @@ import 'package:test/test.dart';
 import 'package:test/test.dart' as testing;
 
 import '../lib/realm.dart';
-import '../lib/src/native/realm_core.dart' as core;
+import '../lib/src/realm_property.dart';
 
-part 'realm_test.gen.dart';
+part 'realm_test.g.dart';
 
 @RealmModel()
 class _Car {
   @PrimaryKey()
-  late String make = "Tesla";
+  late final make = "Tesla";
```

Comment:
Default values for primary keys when we require final is confusing. Should we disallow default values for primary keys?

Reply:
I agree.. I did not rewrite all the tests.

Reply:
Well, then I did anyway :-)

Reply:
Fixed